Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityPrimaryPointerSelector.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Interface used by focus providers to select the primary pointer.
```

Comment:
It would be good to define the primary pointer here.

Reply:
Added a bit more detail. Does that look good?